turtlebot3_gazebo/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(gazebo_ros_pkgs REQUIRED)
-find_package(turtlebot3 REQUIRED)
+find_package(turtlebot3_description REQUIRED)
```

Comment:
Is this line necessary? It seems that turtlebot3_description is used only in launchfiles and is not used by any of the cmake logic

Reply:
I agreed. I've deleted.